packages/@aws-cdk/aws-autoscaling/README.md

```diff
@@ -233,14 +233,58 @@ about allowing connections between resources backed by instances.
 
 To enable the max instance lifetime support, specify `maxInstanceLifetime` property
 for the `AutoscalingGroup` resource. The value must be between 7 and 365 days(inclusive).
-To clear a previously set value, just leave this property undefinied.
+To clear a previously set value, leave this property undefined.
 
 ### Instance Monitoring
 
 To disable detailed instance monitoring, specify `instanceMonitoring` property
 for the `AutoscalingGroup` resource as `Monitoring.BASIC`. Otherwise detailed monitoring
 will be enabled.
 
+### Monitoring Group Metrics
+
+Group metrics are used to monitor group level properties; they describe the group rather than any of its instances (e.g GroupMaxSize, the group maximum size). To enable group metrics monitoring, use the `groupMetricsCollections` property, or the `emitAllGroupMetrics` and `emitGroupMetrics` methods.
```

Comment:
Use a line width of 120 or 150.

Reply:
Should this be changed in the linter?

Reply:
Ideally yes. I've been considering this as well but just haven't gotten around.